relate/settings.py

```diff
@@ -287,4 +287,10 @@
     assert isinstance(
         local_settings["RELATE_EMAIL_APPELATION_PRIORITY_LIST"], list)
 
+# This defaults RELATE_EMAIL_SMTP_ALLOW_NONAUTHORIZED_SENDER to True if
+# not configured in local_settings.py
+RELATE_EMAIL_SMTP_ALLOW_NONAUTHORIZED_SENDER = getattr(
+        local_settings, "RELATE_EMAIL_SMTP_ALLOW_NONAUTHORIZED_SENDER",
+        True)
+
```

Comment:
Why not just set that variable to `True` before the local_settings stuff gets set?